backend/danswer/connectors/notion/connector.py

```diff
@@ -191,6 +192,7 @@ def _read_pages(
                     ],
                     source=DocumentSource.NOTION,
                     semantic_identifier=page_title,
+                    doc_updated_at=datetime.fromisoformat(page.last_edited_time),
```

Comment:
same note on adding a `.astimezone(timezone.utc)` here just for clarity / safety